R/mod-synapse-oauth.R

```diff
@@ -27,7 +27,7 @@ authorization_url <- NULL
     	# start up the first time the app' is run.
     	# From https://stackoverflow.com/questions/54651700/use-python-3-in-reticulate-on-shinyapps-io
     	reticulate::virtualenv_create(envname = venv_folder, python = '/usr/bin/python3')
-    	reticulate::virtualenv_install(venv_folder, packages = c('synapseclient', 'pandas'))
+    	reticulate::virtualenv_install(venv_folder, packages = c('synapseclient<2.8', 'pandas1.5'))
      }
```

Comment:
I think we need to use either 'pandas==1.5' or 'pandas<2.0' from the examples I see on stackoverflow.

Reply:
`    	# start up the first time the app' is run.
    	# From https://stackoverflow.com/questions/54651700/use-python-3-in-reticulate-on-shinyapps-io
    	reticulate::virtualenv_create(envname = venv_folder, python = '/usr/bin/python3')
    	reticulate::virtualenv_install(venv_folder, packages = c('synapseclient<2.8', 'pandas<1.5'))
     }
     reticulate::use_virtualenv(venv_folder, required = T)
  }`

Tom's change